src/aliceVision/sfm/utils/alignment.cpp

```diff
@@ -1036,6 +1036,56 @@ void computeCentersVarCov(const sfmData::SfMData& sfmData, const Vec3 & mean, Ei
     }
 }
 
+void computeNewCoordinateSystemGroundAuto(const sfmData::SfMData& sfmData, Vec3& out_t)
+{
```

Comment:
Add comments describing the steps as in the PR

Reply:
done